PWGCF/Flow/Tasks/flowPtEfficiency.cxx

```diff
@@ -89,8 +146,14 @@ struct flowPtEfficiency {
   }
   PROCESS_SWITCH(flowPtEfficiency, processReco, "process reconstructed information", true);
 
-  void processSim(aod::McCollision const&, soa::SmallGroups<soa::Join<aod::McCollisionLabels, aod::Collisions>> const& collisions, myMcParticles const& mcParticles)
+  void processSim(myMcCollisions::iterator const& collision, aod::BCsWithTimestamps const&, soa::SmallGroups<soa::Join<aod::McCollisionLabels, aod::Collisions>> const& collisions, myMcParticles const& mcParticles)
   {
+    if (cfgSelRunNumberEnabled) {
+      auto bc = collision.bc_as<aod::BCsWithTimestamps>();
```

Comment:
Same here
Also consider that the run number should be the same at detector level as at generator level